pywinauto/unittests/test_uiawrapper.py

```diff
@@ -453,6 +454,29 @@ def test_capture_as_image_multi_monitor(self):
                 self.assertEqual(expected, result)
 
 
+    class UIAWrapperRawViewWalkerTests(UIAWrapperTests):
+
+        """Unit tests for the UIAWrapper class with enabled RawViewWalker"""
+
+        def setUp(self):
+            self.assertEqual(UIAElementInfo.use_raw_view_walker, False)
+            UIAElementInfo.use_raw_view_walker = True
```

Comment:
The same comment about keeping previous value.

Reply:
changed